src/app/how-it-works/page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function HowItWorksPage() {
+    return (
+        <div className='w-full h-screen flex-row-center-center'>How It works Section</div>
```

Comment:
does this conflict with `src/app/how-it-works/page.mxd?`

Reply:
No it will not create any conflict.

Reply:
I don't think it would create any conflict. I will check again.